libraries/card-input-view/src/main/java/com/trendyol/cardinputview/CardInputView.kt

```diff
@@ -108,6 +109,16 @@ class CardInputView : ConstraintLayout {
         binding.executePendingBindings()
     }
 
+    /**
+     *
+     * Clears all input fields' error states
+     * it will immediately sets all fields backgrounds with [CardInputViewState.inputBackgroundDrawable].
+     */
+    fun clearErrors() {
```

Comment:
This function does not clear error states. It just hides.
I suggest renaming. 

Also, we may add a boolean parameter

Reply:
Actually, the view itself does not keep the state of errors instead only showing errors as state in the name of visibility as result of the validation.